packages/track-transcript/src/TranscriptConnected.js

```diff
@@ -92,4 +31,11 @@ const mapStateToProps = (state, ownProps) => ({
   strand: strand(state),
 })
 
-export default connect(mapStateToProps, geneActions)(TranscriptConnected)
+const mapDispatchToProps = dispatch => ({
+  onExonClick: bindActionCreators(geneActions.setCurrentExon, dispatch),
+  onTissueChange: bindActionCreators(geneActions.setCurrentTissue, dispatch),
+  onTranscriptNameClick: bindActionCreators(geneActions.setCurrentTranscript, dispatch),
+  transcriptButtonOnClick: bindActionCreators(geneActions.toggleTranscriptFanOut, dispatch),
+})
```

Comment:
Instead of using a separate component (`TranscriptConnected`) to map props from Redux to `TranscriptTrack`, we can do that directly in `mapDispatchToProps`.

Reply:
Yeah, this is much cleaner